C4/SIP/ILS/Transaction/Checkout.pm

```diff
@@ -100,6 +100,12 @@ sub do_checkout {
                     last;
                 }
                 last;
+            } elsif ($confirmation eq 'TRANSFER') {
```

Comment:
Here you check $needsconfirmation variable but you set the transfer reason in $issuingimpossible so this line of code would never get called. Now that I think maybe it would be good idea to use $needsconfirmation so the librarian can override the block in order to issue the book to a patron.